interview-questions.md

```diff
@@ -10,28 +10,29 @@ Type your initials here to confirm you read the instructions:
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: if , else ,else if are conditional statements that control the output
 
-Researched answer:
+Researched answer: Conditional statements control behavior in JavaScript and determine whether or not pieces of code can run. There are multiple different types of conditionals in JavaScript including: “If” statements: where if a condition is true it is used to specify execution for a block of code.
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: git is the software that you can attatch to your terminal and Github is the website that has a cloud of code stored within it
 
-Researched answer:
+Researched answer: Git is a version control system that allows developers to track changes in their code. GitHub is a web-based hosting service for git repositories.
```

Comment:
With the research answers give a bit more. Here, maybe talk about how changes are tracked via commits.